vite.js

```diff
@@ -82,19 +91,32 @@ module.exports = function style9Plugin(opts = {}) {
         server
       });
     },
-    renderChunk(_, chunk) {
+    async renderChunk(_, chunk) {
       const ids = Object.keys(chunk.modules);
+      // https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/css.ts#L482-L489
       for (const id of ids) {
         if (id.startsWith(VIRTUAL_CSS_NAME)) {
+          // fool the css plugin to generate the css in corresponding chunk
+          const fakeCssId = `${chunk.fileName}.css`;
+          if (cssPlugin && cssPluginPost) {
```

Comment:
does this mean that if there's no cssplugin, the css is removed entirely?

Reply:
Don't worry.vite contains this plugin